### ISSUE TITLE
BTLA v.1.5 Fan Change #2: SWAT 3's Inspired Movement: No more limping and inaccuracy

### ISSUE THREAD

**Issue by TheSuggester** [EFdee-BTLA/SEF_BTLA#22]
[SkeletalRegionInformation.zip](https://github.com/EFdee-BTLA/SEF_BTLA/files/7227822/SkeletalRegionInformation.zip)

NOTE#1: All of these .ini files are intended for v.1.5 **ONLY**, if you choose to use them for later versions, they **MIGHT** work, **BUT**, they also **MIGHT BREAK THE MOD 100%**. Using backups for this mod is **STRONGLY RECOMMENDED**.

NOTE#2: ALL of these .ini files will work in **SINGLEPLAYER (SP)**. **HOWEVER**, for **MULTIPLAYER (MP)**, all players (including the server) need to have the same .ini file equipped. If this is not done, you can't play **MP** with anyone else.
________________________________________________________________________________________________________________________________________
### (INSTRUCTIONS)

To use this, take this .ini file and paste it over the System Directory.

What is shown below should be the ideal location to replace the .ini file.

Your drive (C:,D:,E:, etc.)/SWAT 4/SEF-BTLA/System
________________________________________________________________________________________________________________________________________
### (DESCRIPTION)

This .ini change makes it where all shots to any part of your body does not have the chance to cripple you. In addition, you will no longer suffer any accuracy loss even when shot in your arms.

In other words, with this .ini change, nothing will be able to cause you to limp anymore and nothing will make you inaccurate either.
________________________________________________________________________________________________________________________________________
### (PERSONAL NOTE)

This was inspired by SWAT 3, that has a similar system in where you can't be crippled in the game and that your accuracy stays at 100% no matter how many times you were shot.
________________________________________________________________________________________________________________________________________
Feel free to comment if this idea should be an official part of BTLA!

If you don't like this change, also say so!

Any and all input here will be greatly appreciated!

**Comment by TheSuggester** [comment]
### FEEDBACK FROM RYKER

![FEEDBACK FROM RYKER](https://user-images.githubusercontent.com/38480961/134728904-d831d784-3609-4d1a-94e4-b5af4131a43e.JPG)

**Comment by TheSuggester** [comment]
Mod DB LINK: https://www.moddb.com/games/swat-4-the-stetchkov-syndicate/downloads/btla-v15-fan-change-2-swat-3s-inspired-movement-no-more-limping-and-inaccu
